### PR TITLE
Create review-of-international-studies.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/62595/style-request-review-of-international-studies

(continuation of #2362)

### CODE REVIEW

File: review-of-international-studies.csl [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+      <group delimiter=": ">
+        <text variable="publisher-place"/>
+        <text variable="publisher"/>
+      </group>
+      <date variable="issued">
+        <date-part name="year"/>
+      </date>
+    </group>
+  </macro>
+  <citation et-al-min="6" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">
```
Comment (adam3smith) [inline review-of-international-studies.csl]: did you find examples for these et al. settings? Otherwise I think we should leave them at 3/1 throughout

**Comment by adam3smith** [comment]
Don't merge yet -- I made my changes quickly and w/o testing, want to take another look.

File: review-of-international-studies.csl (hunk 1 above)
Comment (POBrien333) [reply review-of-international-studies.csl]: I just didn't adapt it. Thanks for noticing.

File: review-of-international-studies.csl [hunk 2]
```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" and="text" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Review of International Studies</title>
+    <id>http://www.zotero.org/styles/review-of-international-studies</id>
+    <link href="http://www.zotero.org/styles/review-of-international-studies" rel="self"/>
+    <link href="http://www.zotero.org/styles/urban-studies" rel="template"/>
+    <link href="https://www.cambridge.org/core/services/aop-file-manager/file/57753670dd5679630ae895f8" rel="documentation"/>
```
Comment (rmzelle) [inline review-of-international-studies.csl]: I think it's a little easier if we point to https://www.cambridge.org/core/journals/review-of-international-studies/information/instructions-contributors instead for the "documentation" link. Likely to be a more stable link and it's easier to get to the journal's homepage from there.

Comment (rmzelle) [reply review-of-international-studies.csl]: Otherwise no issues with style metadata or indentation.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**